score/ScoreBonus.py

```diff
@@ -0,0 +1,275 @@
+from nameinternal import get_card_metadata, get_score_bonus
+from collections import Counter, namedtuple
+from save import Savefile
+
+class ScoreBonus():
+    def __init__(self, name: str):
+        self.name = ""
+        self.format_string = ""
```

Comment:
I would probably pass the format string in the constructor (or maybe include it in the full_data.json file), and then have `get_format_string()` do `if self.format_string: return self.format_string.format(self.score_bonus)`.

Actually, what I would do is have them as properties, and store this as `self._format_string`, and have a `format_string` property that calculates stuff on the fly.

Reply:
yep this seems better.